Fixed issue of Spacing above the heading

MouliiiSharma opened the pull request:
PR Description 📜

Issue #147 that was to enhance the view of page by fixing the heading " Help us improve!" is done now
Fixed # <#147>
- [ ] I follow [CONTRIBUTING GUIDELINE](https://github.com/GameSphere-MultiPlayer/Community-Page/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/GameSphere-MultiPlayer/Community-Page/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [ ] I have performed a self-review of my own code or work.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My changes generates no new warnings.
- [ ] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/GameSphere-MultiPlayer/Community-Page/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [ ] I have added screenshot for review.



![image](https://github.com/ChromeGaming/Physi-c-Tech/assets/146819673/0dd8f589-e78a-4cc0-a701-9d430cce9273)

MouliiiSharma commented:
please check this

thevirengarg commented:
@MouliiiSharma Spacing above the heading will be different for every laptop's screen size ratio. Update it in such a way it should be consistent for every screen size.

MouliiiSharma commented:
okay sir

MouliiiSharma commented:
Now I have changed the heading space in a way so that it will be consistent for every screen size and as content was overflowing I have added a scroll bar too. please check this .

https://github.com/ChromeGaming/Physi-c-Tech/assets/146819673/ea214f0c-0510-4efa-846c-3aa967edfe75
